.vscode/settings.json

```diff
@@ -22,5 +22,6 @@
   },
   "[typescriptreact]": {
     "editor.defaultFormatter": "biomejs.biome"
-  }
+  },
+  "vitest.disableWorkspaceWarning": true
```

Comment:
I've set that because Vitest is freaking out a bit in this monorepo and missing quite a few tests, but my feeling is that most test suites are very small and it's acceptable to scan multiple projects in this instance.

Reply:
Yup this is fine